packages/Webkul/Velocity/src/Resources/assets/sass/components/app.scss

```diff
@@ -777,6 +777,7 @@
 
                     .rango-delete {
                         margin-left: -2px;
+                        float: left;
```

Comment:
Should be right in RTL and set properly because this will not work as expected in RTL.

Reply:
done.